client/my-sites/plugins/plugin-settings-link/index.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+
+export default React.createClass( {
+	displayName: 'PluginSettingsLink',
+
+	propTypes: {
+		link: React.PropTypes.string.isRequired,
```

Comment:
`link` sounds a bit ambiguous. maybe something like `linkUrl` would be clearer to someone using this